### PR TITLE
handbrake 1.4.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
Undefined symbols for architecture x86_64:
  "_DADiskCopyDescription", referenced from:
      _mount_get_mountpoint in libbluray.a(libbluray_la-mount_darwin.o)
  "_DADiskCreateFromBSDName", referenced from:
      _mount_get_mountpoint in libbluray.a(libbluray_la-mount_darwin.o)
  "_DASessionCreate", referenced from:
      _mount_get_mountpoint in libbluray.a(libbluray_la-mount_darwin.o)
  "_kDADiskDescriptionVolumePathKey", referenced from:
      _mount_get_mountpoint in libbluray.a(libbluray_la-mount_darwin.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [HandBrakeCLI] Error 1
```

File: Formula/handbrake.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class Handbrake < Formula
   desc "Open-source video transcoder available for Linux, Mac, and Windows"
   homepage "https://handbrake.fr/"
-  url "https://github.com/HandBrake/HandBrake/releases/download/1.3.3/HandBrake-1.3.3-source.tar.bz2"
-  sha256 "218a37d95f48b5e7cf285363d3ab16c314d97627a7a710cab3758902ae877f85"
+  url "https://github.com/HandBrake/HandBrake.git",
```
Comment (carlocab) [inline Formula/handbrake.rb:4]: I don't think we want to switch to a Git clone here. Try adding a `depends_on "libbluray"` instead.

Comment (chenrui333) [reply Formula/handbrake.rb:4]: 👍

**Comment by chenrui333** [comment]
#83421

**Pull request closed without merging**